uint16 speed_command
int16 angle_of_correction
string forward_reverse
